packages/react/src/components/Modal/Modal.test.tsx

```diff
@@ -44,8 +30,67 @@ const render = async (props: Partial<ModalRootProps> = {}) => {
 };
 
 describe('Modal', () => {
+  beforeAll(() => {
+    HTMLDialogElement.prototype.show = vi.fn(function mock(
+      this: HTMLDialogElement,
+    ) {
+      this.open = true;
+    });
```

Comment:
If you declare these mock functions outside we can also use them to assert that they have been called x times :)

Reply:
Will move them to the setup file 😄